scripts/helper/create-cluster.sh

```diff
@@ -23,12 +23,18 @@ if [ ! -e helper/parameters.sh ]; then
 fi
 
 source helper/parameters.sh
+export SANITIZED_OCP_VERSION
+
+function terraform_apply () {
+  cat $WORKSPACE/ocs-upi-kvm/files/site.tfvars.in | envsubst > $WORKSPACE/site.tfvars
```

Comment:
Please follow the indenting scheme of the file -- 8 space tab.   See below also...